app/src/main/java/com/firebase/uidemo/database/ChatActivity.java

```diff
@@ -182,4 +177,12 @@ private void updateUI() {
         mSendButton.setEnabled(isSignedIn());
         mMessageEdit.setEnabled(isSignedIn());
     }
+
+    // TODO remove once arch components are merged into support lib
```

Comment:
Is the idea that eventually an `AppCompatActivity` will automatically contain the registry?